mars.yml

```diff
@@ -0,0 +1,403 @@
+version: 1
```

Comment:
@jelysn can we move this fine within the script folder where the actual script using this file lives?

Reply:
I think this might be useful on other features which require multiple networks setup.

Reply:
@jelysn what I meant is to keep the root of the repo clean, it would be to move this config file within a separate folder as it will used for now only for this purpose

Reply:
Ok

Reply:
Moved config.yml to transferhook scripts directory